state/pg_state_manager.go

```diff
@@ -904,7 +918,7 @@ func (sm *SQLStateManager) BatchUpdateWorkers(updates []Worker) (WorkersList, er
 // Cleanup close any open resources
 //
 func (sm *SQLStateManager) Cleanup() error {
-	return sm.db.Close()
+	return multierr.Combine(sm.db.Close(), sm.readonlyDB.Close())
```

Comment:
@usman-r

Reply:
Just double checked the sqlx api and both return an Close and Combine return an err so 👍